### DISCUSSION TITLE
PB with the new version

### DISCUSSION THREAD

**Discussion by davidclaverie** [RomRider/apexcharts-card#741] [Q&A]
With the new version i have problem 
see pic please
Why?
![ok temp](https://github.com/user-attachments/assets/dfff82e2-db19-4a6d-b857-869e8cdf266c)
![pb temp apex](https://github.com/user-attachments/assets/aed7bd81-3193-41b0-a8eb-a020cae1a4ce)

**Comment by Spl4sh57** [comment]
J'ai exactement le même problème que toi !
I have the same issue.

**Comment by davidclaverie** [comment]
please if you find the solution say to me.

Le mar. 23 juil. 2024 à 10:34, Spl4sh57 ***@***.***> a écrit :

> J'ai exactement le même problème que toi !
> I have the same issue.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/RomRider/apexcharts-card/discussions/741#discussioncomment-10124341>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASTYDJBDGIKMU662LR3H2P3ZNYPTXAVCNFSM6AAAAABLGTHPJGVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMJSGQZTIMI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
__________________________
David CLAVERIE​
Mobile / WhatsApp: +241 (0)66 26 25 00 <+241%20066%2026%2025%2000>
P Pensez a l'environnement avant d imprimer cet email . Please consider the
environment before printing this email
Les informations contenues dans ce message sont confidentielles et à
l'attention exclusive de son (ou ses) destinataire(s). Si vous n'êtes pas
destinataire de ce message, sachez que toute utilisation édition,
modification ou diffusion de ce message est strictement interdite. Si vous
avez reçu ce message par erreur, merci de nous en informer immédiatement
par téléphone ou email et de le supprimer de votre système.

**Comment by Spl4sh57** [comment]
For me, before : 
![a4f4bcff9612a5e6b1999aff4230ff4cf6bfc65b](https://github.com/user-attachments/assets/0e86c93c-112e-4d0a-a5a1-351099e05a90)

after : 
![f519c315d39c5eaa8f8ccdcde04f9e662bee116f](https://github.com/user-attachments/assets/548e0d29-5109-4135-bc1e-8f0baac37844)

**Comment by davidclaverie** [comment]
yes same thing i think

Le mar. 23 juil. 2024 à 13:14, Spl4sh57 ***@***.***> a écrit :

> For me, before :
> a4f4bcff9612a5e6b1999aff4230ff4cf6bfc65b.png (view on web)
> <https://github.com/user-attachments/assets/0e86c93c-112e-4d0a-a5a1-351099e05a90>
>
> after :
> f519c315d39c5eaa8f8ccdcde04f9e662bee116f.png (view on web)
> <https://github.com/user-attachments/assets/548e0d29-5109-4135-bc1e-8f0baac37844>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/RomRider/apexcharts-card/discussions/741#discussioncomment-10125870>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASTYDJDCEKY4LTB4PV7RCHTZNZCLJAVCNFSM6AAAAABLGTHPJGVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMJSGU4DOMA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
__________________________
David CLAVERIE​
Mobile / WhatsApp: +241 (0)66 26 25 00 <+241%20066%2026%2025%2000>
P Pensez a l'environnement avant d imprimer cet email . Please consider the
environment before printing this email
Les informations contenues dans ce message sont confidentielles et à
l'attention exclusive de son (ou ses) destinataire(s). Si vous n'êtes pas
destinataire de ce message, sachez que toute utilisation édition,
modification ou diffusion de ce message est strictement interdite. Si vous
avez reçu ce message par erreur, merci de nous en informer immédiatement
par téléphone ou email et de le supprimer de votre système.

**Comment by darkvanloc** [comment]
Même problème que vous 

![image](https://github.com/user-attachments/assets/ff199293-d9f0-4372-b897-5967c63264ae)

**Comment by Spl4sh57** [comment]
I've gone back to the old version 2.0.4

**Comment by davidclaverie** [comment]
yes mee to i just try this morning and working Great now  with the
version2.0.4 exact.

Le mer. 24 juil. 2024 à 15:19, Spl4sh57 ***@***.***> a écrit :

> I've gone back to the old version 2.0.4
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/RomRider/apexcharts-card/discussions/741#discussioncomment-10138541>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASTYDJHXQXLIYGRN2CLGHNDZN6ZVPAVCNFSM6AAAAABLGTHPJGVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMJTHA2TIMI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
__________________________
David CLAVERIE​
Mobile / WhatsApp: +241 (0)66 26 25 00 <+241%20066%2026%2025%2000>
P Pensez a l'environnement avant d imprimer cet email . Please consider the
environment before printing this email
Les informations contenues dans ce message sont confidentielles et à
l'attention exclusive de son (ou ses) destinataire(s). Si vous n'êtes pas
destinataire de ce message, sachez que toute utilisation édition,
modification ou diffusion de ce message est strictement interdite. Si vous
avez reçu ce message par erreur, merci de nous en informer immédiatement
par téléphone ou email et de le supprimer de votre système.
